locales/en/plugin__forklift-console-plugin.json

```diff
@@ -266,7 +266,6 @@
   "Hide variables": "Hide variables",
   "Hooks": "Hooks",
   "Hooks (optional)": "Hooks (optional)",
-  "Hooks for virtualization": "Hooks for virtualization",
```

Comment:
How does removing trans happen when removing indexes?

Reply:
We had 2 modules with extensions for hooks and migrations that seemed unused, so I removed them as well